conf/app.ini

```diff
@@ -203,7 +203,8 @@ IMPORT_LOCAL_PATHS = false
 ; Whether to allow signin in via OpenID
 ENABLE_OPENID_SIGNIN = true
 ; Whether to allow registering via OpenID
-ENABLE_OPENID_SIGNUP = true
+; Do not include to rely on DISABLE_REGISTRATION setting
+;ENABLE_OPENID_SIGNUP = true
```

Comment:
@bkcsoft I just realized it doesn't take write access to `custom/` to do what you're asking, which is basically still support the old variable in custom if present. Sounds a bit complex though, is that still worth the advantage being the possibility to write `DISABLE_OPENID_REGISTRATION = %DISABLE_REGISTRATION` in `custom/conf/app.ini` ?